lib/utilityservice.js

```diff
@@ -410,6 +414,45 @@ UtilityService.prototype.checkLogin = async function() {
     }
 };
 
+UtilityService.prototype.sfdx = async function(command, options) {
+    var org = {
+        getUsername: function() {
+            if(options.sessionId && options.instanceUrl){
+                return options.sessionId;
+            }
+            return options.targetusername;
+        }
+    };
+
+    var flags = options;
+
+    flags.quiet = VlocityUtils.quiet;
+    flags.json = VlocityUtils.quiet || !VlocityUtils.showLoggingStatements;
+
+    let allCacheKeys = Object.keys(require.cache);
+
+    for (let key of allCacheKeys) {
```

Comment:
What is this for? what cache are we suspecting?

Reply:
It was there with older method also , i guess  this can be removed, removing it

Reply:
If it was there, Let's not remove it. We are not sure of consequences.

Reply:
removed it tests are passing, for now anyways we are executing things on shell, this cache will not have any impact